### DISCUSSION TITLE
img2img inpaint: possible bug - it seems to not forget the previous mask from last inpainting session

### DISCUSSION THREAD

**Discussion by Tillerz** [vladmandic/sdnext#754] [Q&A] [answered]
If you go to img2img inpaint and mask an area that shall get updated, it sometimes applies the (not visible anymore) mask being used last time, too. Looks like the mask is still there, not visible, and if you then mask something, this just gets added to the existing mask. The 'clear' button in the mask area fixes that, though.

**Comment by vladmandic** [comment, marked as answer]
There is an open issue for thar.

**Reply by Tillerz** [reply]
Ow, okay. :)
